public/components/model_list/model_table.tsx

```diff
@@ -145,15 +155,25 @@ export function ModelTable(props: ModelTableProps) {
             name: 'Prevew',
             description: 'Preview model group',
             type: 'icon',
-            icon: 'boxesHorizontal',
+            icon: 'plusInCircle',
             onClick: ({ name }) => {
               onModelNameClick(name);
             },
           },
+          // TODO: add a new task to update after design completed
+          {
+            name: 'Delete',
+            description: 'Delete this',
+            type: 'icon',
+            icon: 'trash',
+            onClick: ({ name, ...args }) => {
```

Comment:
Prefer change to 
```tsx
onClick: ({name, deployed_versions: deployedVersions})=>{
        onModelDeleteClick(name, deployedVersions);
}
```
Since we need to display model link in the confirmation modal, do we need to pass id too?